CHANGES.md

```diff
@@ -85,6 +85,12 @@ OpenSSL 3.5
 
    *David von Oheimb*
 
+ * Added support for central key generation in CMP.
+
+   This work was sponsored by Siemens AG.
+
+    *Rajeev Ranjan*
+
```

Comment:
Please squash your commits regarding `fixup! CMP: add documentation`
as you in between accidentally deleted `CHANGES.md`.